src/components/players/list/index.js

```diff
@@ -39,14 +43,14 @@ class List extends React.Component {
                     <div className={styles.imageContainer}>
                         <img 
                             src={`https://ak-static.cms.nba.com/wp-content/uploads/headshots/nba/${selectedTeam.teamId}/2019/260x190/${player.person_id}.png`} 
-                            alt="Player Headshot" 
+                            alt="player headshot" 
                             onError={(e) => e.target.src = placeholderImg}
                         />
                     </div>
-                    <div className={styles.imageLine} />
-                    <div className={styles.detailsContainer}>
-                        <div className={styles.number}>{player.jersey_number}</div>
-                        <div className={styles.details}>
+                    <div style={{ borderBottom: `2px solid ${teamColor}`}} className={styles.imageLine} />
```

Comment:
You can just write this as (also notice no need for string literals):
```
 <div style={{ borderColor: teamColor }} className={styles.imageLine} />
```

and then in `.imageLine` css just do:

```
  .imageLine {
    border-bottom: 2px solid transparent;
    width: 100%;
    margin-top: 4px;
  }
```